doc/fuzzing.md

```diff
@@ -56,7 +56,6 @@ AFLOUT=$PWD/outputs
 Example inputs are available from:
 
 - https://download.visucore.com/bitcoin/bitcoin_fuzzy_in.tar.xz
```

Comment:
Could add the hash of the file and a mirror url to avoid this happening in the future?